### PR TITLE
lint improvements

### PR DESCRIPTION
**Pull request by tmc**
This chips away at outstanding lint issues.

### CODE REVIEW

File: src/sybil/column_store_io.go [hunk 1]
```diff
@@ -659,12 +659,6 @@ func (tb *TableBlock) unpackSetCol(dec FileDecoder, info SavedColumnInfo) {
 		}
 	} else {
 		for r, v := range into.Values {
-			curSet, ok := records[r].SetMap[colID]
-			if !ok {
-				curSet = make(SetField, 0)
-				records[r].SetMap[colID] = curSet
-			}
-
```
Comment (okayzed) [inline src/sybil/column_store_io.go]: we should likely test this by inducing a high cardinality set codepath. can do that by creating a set column where each set includes a string that is counting upwards to 65,000.

File: src/sybil/table.go [hunk 2]
```diff
@@ -26,13 +26,13 @@ type Table struct {
...
-	keyStringIdLookup map[int16]string
-	valStringIdLookup map[int32]string
+	keyStringIDLookup map[int16]string
+	valStringIDLookup map[int32]string
 
 	// This is used for join tables
 	joinLookup map[string]*Record
 
-	stringIdM *sync.RWMutex
+	stringIDM *sync.RWMutex
```
Comment (okayzed) [inline src/sybil/table.go]: stringIDM is weird, imo. dunno what's better, though - maybe something like... stringID_m?

Comment (tmc) [reply src/sybil/table.go]: yeah agree. stringIdMu ok? I can rename other mutexes to follow that pattern.

Comment (tmc) [reply src/sybil/table.go]: stringIDMu *

**Pull request merged**